src/forms/EditListingDescriptionForm/CustomCertificateSelectFieldMaybe.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { required } from '../../util/validators';
+import { FieldSelect } from '../../components';
+
+import css from './EditListingDescriptionForm.css';
+
+const CustomCertificateSelectFieldMaybe = props => {
+  const { name, id, certificate, intl } = props;
+  const certificateLabel = intl.formatMessage({
+    id: 'EditListingDescriptionForm.certificateLabel',
+  });
+  const certificatePlaceholder = intl.formatMessage({
+    id: 'EditListingDescriptionForm.certificatePlaceholder',
+  });
+  const certificateRequired = required(
+    intl.formatMessage({
+      id: 'EditListingDescriptionForm.certificateRequired',
+    })
+  );
+  return certificate ? (
+    <FieldSelect
+      className={css.certificate}
+      name={name}
+      id={id}
+      label={certificateLabel}
+      validate={certificateRequired}
```

Comment:
I don't think this should be required field.
And if you remove this validate prop, you could also remove certificatePlaceholder option. That way the first certificate becomes selected on submit call if the user has not touched the select field.